package.json

```diff
@@ -51,11 +60,15 @@
     "test:lint": "eslint bin/marked .",
     "test:redos": "eslint --plugin vuln-regex-detector --rule '\"vuln-regex-detector/no-vuln-regex\": 2' lib/marked.js",
     "bench": "node test --bench",
-    "lint": "eslint --fix bin/marked .",
-    "build": "uglifyjs lib/marked.js -cm  --comments /Copyright/ -o marked.min.js",
     "preversion": "npm run build && (git diff --quiet || git commit -am 'minify')"
   },
   "engines": {
     "node": ">=0.10.0"
-  }
+  },
+  "files": [
+    "bin/**",
+    "lib/**",
+    "man/**",
+    "README.md"
```

Comment:
I see now, this is a better solution than npmignore

Reply:
Note to self: this needs `src/**` adding for compatibility with browsers importing the ESM source, though.